gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GcsJavaClientImpl.java

```diff
@@ -56,6 +77,30 @@ public GoogleCloudStorageOptions getOptions() {
   @Override
   public WritableByteChannel create(StorageResourceId resourceId, CreateObjectOptions options)
       throws IOException {
+    if (storageOptions.isGrpcEnabled()) {
```

Comment:
why we need this. I thought GCS client support both gRPC and JSON and one can provide a flag to pick the underlying API?

Reply:
We are enabling Json by default. There are some features which are not supported OR underdevelopment in Json. As part of this migration we are focusing on gRPC and Json will follow later.